### DISCUSSION TITLE
[Unstable] Separatist medic uniforms have swapped inventory icons

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#14443] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The medicseparatistsuniform1 (brown) has the green uniform icon while medicseparatistsuniform2 (green) has the brown uniform icon
![image](https://github.com/user-attachments/assets/55e318c8-2c85-4b21-b03f-469292bafed0)

### Reproduction steps

Compare medicseparatistsuniform1 and medicseparatistsuniform2 sprites and inventory icons

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.6.2.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
The issue has now been fixed in our private repo, fix coming up in the next unstable build.
